package.json

```diff
@@ -89,8 +87,7 @@
     "vite-plugin-svgr": "^4.2.0",
     "vite-tsconfig-paths": "^4.2.3",
     "vitest": "^1.1.3",
-    "vitest-dom": "^0.1.1",
```

Comment:
`vitest-dom` is used in `src/setup-tests.ts`, though it seems that the lib has been deprecated now. Before removing it, we should find a proper replacement for it (or check that it's not really adding anything to the project)

Reply:
`vitest-dom` is not deleted, it was just modified the comma of the line

Reply:
sorry 🤦🏻‍♂️